doc/superbuild/source/developers/style_guide/SourceCode.rst

```diff
@@ -355,24 +355,6 @@ not adhere to all of these rules.
    ``sunindextype`` for it. Instead use the appropriate integer type (e.g., ``uint64_t``) directly.
    Do not use ``sunindextype`` for counters either.
 
-#. ``SUNLogger`` statements must be in the format:
```

Comment:
Add this bullet back but in brief (i.e. without the examples),  and reference the logging section for the details.